app/views/adoptable_pets/show.html.erb

```diff
@@ -114,47 +102,47 @@
                   <% if AdopterApplication.adoption_exists?(current_user.adopter_account.id, @pet.id) %>
                     <div class='d-flex align-items-center mt-3'>
                       <h4 class="me-2">
-                        Application <%= @adoption_application.status.titleize || " submitted" %>
+                        <%= "#{t(".status." + @adoption_application.status)}" %>
```

Comment:
This dot-accessing variable thing is so cool!

Reply:
Oh but did we leave out the conditional where it can have `" submitted"`?

Reply:
Yeah, I dont think thats a condition we'll realistically run into, I can add it back in just in case